content/id/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Garbage Collection
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+Peran dari _garbage collector_ Kubernetes adalah untuk menghapus objek tertentu yang sebelumnya mempunyai pemilik, tetapi tidak lagi mempunyai pemilik.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Pemilik dan _dependent_
+
+Beberapa objek Kubernetes adalah pemilik dari objek lainnya. Sebagai contoh, sebuah ReplicaSet adalah pemilik dari sekumpulan Pod. Objek-objek yang dimiliki disebut _dependent_ dari objek pemilik. Setiap objek _dependent_ memiliki sebuah kolom `metadata.ownerReferences` yang menunjuk ke objek pemilik.
+
+Terkadang, Kubernetes menentukan nilai dari `ownerReference` secara otomatis. Sebagai contoh, ketika kamu membuat sebuah ReplicaSet, Kubernetes secara otomatis akan menentukan tiap kolom `ownerReference` dari tiap Pod di dalam ReplicaSet. Pada versi 1.8, Kubernetes secara otomatis menentukan nilai dari `ownerReference` untuk objek yang diciptakan atau diadopsi oleh ReplicationController, ReplicaSet, StatefulSet, DaemonSet, Deployment, Job dan CronJob.
+
+Kamu juga bisa menspesifikasikan hubungan antara pemilik dan _dependent_ dengan cara menentukan kolom `ownerReference` secara manual.
+
+Berikut adalah berkas untuk sebuah ReplicaSet yang memiliki tiga Pod:
+
+{{< codenew file="controllers/replicaset.yaml" >}}
+
+Jika anda membuat ReplicaSet tersebut dan kemudian melihat metadata Pod, kamu akan melihat kolom OwnerReferences:
+
+```shell
+kubectl apply -f https://k8s.io/examples/controllers/replicaset.yaml
+kubectl get pods --output=yaml
+```
+
+Keluaran menunjukkan bahwa pemilik Pod adalah sebuah ReplicaSet bernama `my-repset`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  ...
+  ownerReferences:
+  - apiVersion: apps/v1
+    controller: true
+    blockOwnerDeletion: true
+    kind: ReplicaSet
+    name: my-repset
+    uid: d9607e19-f88f-11e6-a518-42010a800195
+  ...
+```
+{{< note >}}
+Referensi pemilik lintas _namespace_ tidak diperbolehkan oleh desain. Artinya:
+1) _Namespace-scoped dependent_ hanya bisa menspesifikasikan pemilik jika berada di _namespace_ yang sama, dan pemilik bersifat _cluster-scoped_. 
```

Comment:
_Namespace-scoped dependent_ -> Dependen dengan cakupan namespace
pemilik bersifat _cluster-scoped_ -> pemilik memiliki cakupan kluster

Reply:
fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305